types/lodash/index.d.ts

```diff
@@ -13908,10 +13908,15 @@ declare namespace _ {
          * _.pick(object, ['a', 'c']);
          * // => { 'a': 1, 'c': 3 }
          */
-        pick<T extends object>(
+        pick<T extends object, U extends keyof T>(
+            object: T | null | undefined,
```

Comment:
How can this take `undefined` as input but return a defined output?

Reply:
`_.pick(undefined)` returns an empty object.

Reply:
The type will still be wrong because `_.pick(undefined, ["x"])` will not have an `x` property. It would make sense to only accept `undefined` in the second overload that returns `Partial`.